File: Folder Java/Kode/TokoKue.java
// TokoKue.java
class TokoKue extends TempatUsaha {
    private String namaProduk;
    private String jenis;
    private int stok;

    public TokoKue() {}
    
    public TokoKue(String namaTempat, String noIzin, String kategoriUsaha, String lokasi, String deskripsi,
                   String namaProduk, String jenis, int stok) {
        super(namaTempat, noIzin, kategoriUsaha, lokasi, deskripsi);
        this.namaProduk = namaProduk;
        this.jenis = jenis;
        this.stok = stok;
    }
}

File: Folder Python/Kode/TokoKue.py
# TokoKue.py
from TempatUsaha import TempatUsaha

class TokoKue(TempatUsaha):
    def __init__(self, nama_tempat="", no_izin="", kategori_usaha="", lokasi="", deskripsi="",
                 nama_produk="", jenis="", stok=0):
        super().__init__(nama_tempat, no_izin, kategori_usaha, lokasi, deskripsi)
        self.__nama_produk = nama_produk
        self.__jenis = jenis
        self.__stok = stok